src/react-app-env.d.ts

```diff
@@ -1 +1,6 @@
-/// <reference types="react-scripts" />
+// / <reference types="react-scripts" />
+
+type Post = {
+  title: string;
+  body: string;
+};
```

Comment:
Please add missing typing for comments and users, just like Post. 
The second option is to use interface.
You can check differences here:
https://www.typescriptlang.org/play#example/types-vs-interfaces